src/index.js

```diff
@@ -2,0 +1,43 @@
+// 'use strict';
+
+// import express from 'express';
+// import cors from 'cors';
+
+// const app = express();
+
+// app.use(cors({ origin: '*' }));
+// app.use(express.json());
+
+// const messages = [];
+
+// app.get('/', (req, res) => {
+//   res.send('Hello World');
+// });
+
+// app.post('/message', (req, res) => {
```

Comment:
The POST route for adding messages is commented out. Uncomment these lines to handle incoming messages.